evm_loader/cli/src/rpc/db_call_client.rs

```diff
@@ -59,19 +59,21 @@ impl CallDbClient {
 
     fn get_account_at_(&self, pubkey: &Pubkey) -> Result<Option<Account>, Error> {
         let pubkey_bytes = pubkey.to_bytes();
+        let slot: i64 = self.slot.try_into().expect("slot < i64::MAX");
```

Comment:
This error message might confuse user. It's better to write something like "slot number must fit into i64".

Reply:
Outdated